libbpfgo/libbpfgo.go

```diff
@@ -790,6 +814,7 @@ func (pb *PerfBuffer) Close() {
 	}
 	pb.Stop()
 	C.perf_buffer__free(pb.pb)
+	eventChannels.Remove(pb.slot)
```

Comment:
Didn't `rb.Stop()` take care of this already?

Reply:
I've removed the one in Stop